docs/install.md

```diff
@@ -366,6 +366,10 @@ use of custom tasks in pipelines.
 most stable features to be used. Set it to "alpha" to allow alpha
 features to be used.
 
+- `scope-when-expressions-to-task`: set this flag to "true" to scope `when` expressions to guard a `Task` only. Set it
+  to "false" to guard a `Task` and its dependent `Tasks`. It defaults to to "false". For more information, see [guarding
```

Comment:
nit: extra to in - It defaults to to "false".

Reply:
fixed it, thanks :)